requirements.txt

```diff
@@ -0,0 +1,16 @@
+appdirs==1.4.3
+asgiref==3.2.7
+attrs==19.3.0
+beautifulsoup4==4.9.0
+black==19.10b0
+click==7.1.2
+Django==3.0.6
+djangorestframework==3.11.0
+pathspec==0.8.0
+pyparsing==3.0.0a1
+pytz==2020.1
+regex==2020.5.7
+soupsieve==2.0
+sqlparse==0.3.1
+toml==0.10.0
+typed-ast==1.4.1
```

Comment:
Wouldn't it be better to make these version ranges so that a minor bug fix version doesn't cause issues?

Reply:
I've changed them all to the compatible version specifier.